src/App.js

```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <div>
+    <Calculator />
+  </div>
+);
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a requirment of for this project, change your `App.js` component which is currently a functional component to a class-based component, something like so: 

```
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
```
Please kindly fix that.